attestation/src/extensions/pkix.rs

```diff
@@ -0,0 +1,83 @@
+//! PKIX X.509 Certificate Extensions (RFC 5280)
+
+pub mod name;
+
+mod authkeyid;
+
+use crate::attr::AttributeTypeAndValue;
+
+use const_oid::{AssociatedOid, ObjectIdentifier};
+
+pub use const_oid::db::rfc5280::{
+    ID_CE_INHIBIT_ANY_POLICY, ID_CE_ISSUER_ALT_NAME, ID_CE_SUBJECT_ALT_NAME,
+    ID_CE_SUBJECT_DIRECTORY_ATTRIBUTES, ID_CE_SUBJECT_KEY_IDENTIFIER,
+};
+
+use der::asn1::{OctetStringRef, SequenceOf};
+
+/// SubjectKeyIdentifier as defined in [RFC 5280 Section 4.2.1.2].
+///
+/// ```text
+/// SubjectKeyIdentifier ::= KeyIdentifier
+/// ```
+///
+/// [RFC 5280 Section 4.2.1.2]: https://datatracker.ietf.org/doc/html/rfc5280#section-4.2.1.2
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub struct SubjectKeyIdentifier<'a>(pub OctetStringRef<'a>);
+
+impl<'a> AssociatedOid for SubjectKeyIdentifier<'a> {
+    const OID: ObjectIdentifier = ID_CE_SUBJECT_KEY_IDENTIFIER;
+}
+
+impl_newtype!(SubjectKeyIdentifier<'a>, OctetStringRef<'a>);
+
+// /// SubjectAltName as defined in [RFC 5280 Section 4.2.1.6].
+// ///
+// /// ```text
+// /// SubjectAltName ::= GeneralNames
+// /// ```
+// ///
+// /// [RFC 5280 Section 4.2.1.6]: https://datatracker.ietf.org/doc/html/rfc5280#section-4.2.1.6
+// #[derive(Clone, Debug, Default, PartialEq, Eq)]
+// pub struct SubjectAltName<'a>(pub name::GeneralNames<'a>);
```

Comment:
Why is this and `IssuerAltName` commented out?

Reply:
Left overs, they're uncommented now.